### ISSUE TITLE
PanelMenu: `aria-expanded` attribute missing when PanelMenu items are collapsed

### ISSUE THREAD

**Issue by blesildaramirez** [primefaces/primevue#6430]
### Describe the bug

When using the `PanelMenu` component and passing the `:expandedKeys` prop to control the expanded state of menu items, I noticed an inconsistency with the `aria-expanded` attribute for the header elements.

The div element with `data-pc-section="header"` includes the `aria-expanded` attribute when the menu item is expanded, and its value is set to `"true"`. However, when the item is collapsed, the `aria-expanded` attribute is removed entirely, instead of being updated to `"false"`.

### Reproducer

https://primevue.org/panelmenu/#controlled

### PrimeVue version

4.x

### Vue version

3.x

### Language

ALL

### Build / Runtime

Vite

### Steps to reproduce the behavior

1. Go to official demo with `:expanded-keys` passed as a prop https://primevue.org/panelmenu/#controlled
2. Inspect the element `data-pc-section="header"` with `aria-label="Users"`, this is the first item from the panel menu example.
3. Expand "Users" item, the `aria-expanded="true"` is added in the element attributes.
4. Collapse "Users" item. the `aria-expanded` disappears. The `data-p-active` is also removed here, it stays on some of the examples so I think it should stay as well, having the value "false" when the item is collapsed.

### Expected behavior

- The `aria-expanded` attribute should remain on the `header` element, regardless of whether the item is expanded or collapsed.
- When the item is expanded, `aria-expanded` should be `"true"` and `data-p-active` should also be `"true"`.
- When the item is collapsed, `aria-expanded` should be `"false"` and `data-p-active` should also be `"false"`.

**Comment by tugcekucukoglu** [comment]
We think this is not a bug, so we are temporarily closing it. We can move it to Discussions.

**Comment by jardakotesovec** [comment]
Hi @tugcekucukoglu,
thank you for checking.  

Would like to support  this bug report, as it breaks the accessibility for screen readers. Which was main the reason why we wanted to rely on the primevue library.

We discovered it through actual screen reader testing. While tracking down the issue with our own usage of the PanelMenu we discovered that it goes wrong when used in the controlled mode, which is easy to reproduce on your own demo - [controlled demo](https://primevue.org/panelmenu/#controlled).

Its essential that aria-expanded is correctly applied. 

If you can't reproduce it let us know, but its important bug from accessibility point of view.
